lib/backend/k8s/resources/ipam_handle.go

```diff
@@ -35,6 +35,7 @@ import (
 const (
 	IPAMHandleResourceName = "IPAMHandles"
 	IPAMHandleCRDName      = "ipamhandles.crd.projectcalico.org"
+	DeletedFinalizer       = "deleted.projectcalico.org"
```

Comment:
I think if I saw this finalizer in `kubectl get -o yaml` output I'd think that it meant that the handle had been deleted already.  How about `ipam.projectcalico.org`?

Reply:
Yes, very much agree